src/containers/BugzillaGraph/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import withErrorBoundary from '../../hocs/withErrorBoundary';
+import { withErrorBoundary } from '../../vendor/errors';
```

Comment:
I must be too out of the groove but I can't figure out why this change would be needed.
I would have assumed that using `import foo from '/path/to/foo'` would work.

Reply:
When I moved `../hocs/withErrorBoundary` to `../vendor/errors` I added more exportable items.